RoRnet.py

```diff
@@ -83,15 +83,18 @@ def commandName(cmd):
 
 def processCharacterAttachData(data):
 	s = charAttach_data_t()
-	s.command, s.enabled, s.source_id, s.stream_id, s.position = struct.unpack('i?3i', data)
+	s.command, s.source_id, s.stream_id, s.position = struct.unpack('4i', data)
 	return s
 
 def processCharacterPosData(data):
 	s = charPos_data_t()
 	if RORNET_VERSION == "RoRnet_2.34":
-		s.command, s.pos.x, s.pos.z, s.pos.y, s.rot.x, s.rot.y, s.rot.z, s.rot.w, s.animationMode, s.animationTime = struct.unpack('i7f28sf', data)
+		unpacked = struct.unpack("i7f28sf", data)
+                s.command, s.pos.x, s.pos.z, s.pos.y, s.rot.x, s.rot.y, s.rot.z, s.rot.w, s.animationMode, s.animationTime = unpacked
 	else: # RoRnet_2.35 and later
-		s.command, s.pos.x, s.pos.z, s.pos.y, s.rot.x, s.rot.y, s.rot.z, s.rot.w, s.animationMode, s.animationTime = struct.unpack('i7f255sf', data)
+		unpacked = struct.unpack("i5f10s", data)
+                s.command, s.pos.x, s.pos.z, s.pos.y = unpacked[:4]
```

Comment:
The ordering is wrong here. It should be
``` python
                s.command, s.pos.x, s.pos.y, s.pos.z = unpacked[:4]
```

Reply:
Already done